### PR TITLE
Expose resampling for overviews in cogification process

### PR DESCRIPTION
**Pull request by zigaLuksic**

### CODE REVIEW

File: eogrow/utils/map.py [hunk 1]
```diff
@@ -64,6 +69,8 @@ def cogify(
     :param nodata: value to be treated as nodata, default value is None
     :param dtype: output type of the in the resulting tiff, default is None
     :param overwrite: If True overwrite the output file if it exists.
+    :param resampling: The resampling method used to produce overviews. The defaults (when using None) are CUBIC for
```
Comment (mlubej) [inline eogrow/utils/map.py:72]: I'm not sure this is used anywhere.

Comment (zigaLuksic) [reply eogrow/utils/map.py:72]: what you actually meant to say is "Hey you dork, you forgot to delete line 85, get your shit together"

fixed

Comment (zigaLuksic) [reply eogrow/utils/map.py:72]: i'll also deal with the defaults in a second

Comment (zigaLuksic) [reply eogrow/utils/map.py:72]: the resampling is now not passed if it's set to `None` to let the defaults work their magic.

**Review by mlubej — APPROVED** [review]

**Pull request merged**